src/directory/directory.mjs

```diff
@@ -1445,7 +1445,7 @@ export const directory = {
           {
             title: 'Get file properties',
             route: '/lib-v1/storage/get-properties',
-            filters: ['flutter', 'js']
+            filters: ['js']
```

Comment:
This page should not exist for flutter lib-v1, current docs: https://docs.amplify.aws/lib-v1/storage/get-properties/q/platform/flutter/